### PR TITLE
Be9

### PR DESCRIPTION
**Pull request by jasper-howard**
ticket 9. GET /api/articles/:article_id/comments

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -169,3 +169,55 @@ describe("6. GET /api/users", () => {
...
+      });
+  });
+  test("should respond with 404 not found when given valid but non existent article_id", () => {
+    return request(app)
+      .get("/api/articles/100/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
```
Comment (HeatherEver) [inline __tests__/api.test.js]: please add a test for if an article has no comments

**Review by HeatherEver — APPROVED** [review]
nice work, just one test to add!

File: __tests__/api.test.js [hunk 2]
```diff
@@ -68,6 +68,24 @@ describe("4. GET /api/articles/:article_id", () => {
...
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          comment_count: 0,
+        };
+
+        expect(body.article).toMatchObject(objCorrect);
+      });
+  });
```
Comment (HeatherEver) [inline __tests__/api.test.js:88]: this test should be in the /api/articles/2/comments endpoint - it should be a status 200 and return and empty array when trying to access an article with no comments

**Review by HeatherEver — COMMENTED** [review]
Sorry Jasper I possibly wasn't clear enough when mentioned to add that test! I hope you understand what I mean now - if not please drop me a message on slack :)

**Comment by cjplatten** [comment]
looks good!

**Pull request merged**